ios/engine/KMEI/KeymanEngine/Classes/Extension/String+Helpers.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  String+Helpers.swift
+//  KeymanEngine
+//
+//  Created by Gabriel Wong on 2017-10-12.
+//  Copyright © 2017 SIL International. All rights reserved.
+//
+
+extension String {
+  var hasFontExtension: Bool {
+    return self.hasSuffix(".ttf") || self.hasSuffix(".otf")
+  }
```

Comment:
Oh, so we made these nifty helpers I saw frequently in Manager.swift.  Nice.

That being said...  what happened to the original `stringWithCharacter` function from the original NSString+Helpers.m from Obj-C?  Is that automatically handled by Swift now, hence its elimination?

Reply:
`stringWithCharacter` was unused so it was removed.